machine/v1alpha1/types_openstack.go

```diff
@@ -356,6 +381,7 @@ type RootVolume struct {
 	// diskSize specifies the size, in GB, of the created root volume.
 	Size int `json:"diskSize,omitempty"`
 	// availabilityZone specifies the Cinder availability where the root volume will be created.
+	// It is an error to set both this and `StorageAvailabilityZone` in `FailureDomain`.
```

Comment:
Use the json names in this text as that's what a user will expect to see, they won't know the Go names